### ISSUE TITLE
Instantiating multiple Views with ContainerSurface cannot set Surface content on master

### ISSUE THREAD

**Issue by daKmoR** [dmvaldman/samsara#44]
Using the latest master branch I get
ElementOutput.js:150 Uncaught TypeError: Cannot set property 'innerHTML' of undefined

```js
var MySubView = View.extend({
	initialize: function() {
		var container = new ContainerSurface({
			size: [undefined, undefined]
		});

		var background = new Surface({
			content: 'info',
			size:    [undefined, undefined]
		});
		container.add(background);
		this.add(container);
	}
});

var mySubView = new MySubView();
var mySubView2 = new MySubView();

var context = new Context();
context.add(mySubView);
context.add(mySubView2);
context.mount(document.body);
```

PS: on the release version it works just fine (e.g. http://codepen.io/anon/pen/wGJwjL?editors=0010)
PPS: if there is no ContainerSurface it also works just fine

**Comment by dmvaldman** [comment]
That's a bug, thanks for digging into it! This commit should fix this: 170efda065cc067dc181b5a3fd18db257cef1c4a

**Comment by daKmoR** [comment]
yes :) works just fine now :) thx
